docs/components/modeler/web-modeler/advanced-modeling/manage-connector-templates.md

```diff
@@ -0,0 +1,103 @@
+
+   The anatomy of the editor interface is as follows:
+
+   - In the **breadcrumbs bar**, you can rename your template by clicking the chevron next to the template name. Note that you cannot change the name of your template in the template json, but only with this action.
+   - On the left, you see the **template JSON editor**. Here, you describe the actual template specification. The specification follows the [most recent element template schema](https://github.com/camunda/element-templates-json-schema). The editor is divided into two sections: a read-only section, containing the schema reference, the template name, and the template id, and a editable section, where you can [write your template specification](/components/modeler/desktop-modeler/element-templates/defining-templates.md).
+   - On the right, you see the live **Visual Preview**. The live preview shows how the properties panel will look when you apply the template to an element. It automatically updates on every valid change, and reflects the latest valid state of the template. The preview allows you to interactively check your template before publishing, including its usability.
+   - In the upper left, you can **Upload an icon** for your template. You can upload any image file you want, however we recommend to use squared SVG graphics. The icons get rendered 18x18 pixels in the element on the modeling canvas, and 32x32 pixels in the properties panel.
+
+   On every valid change, the template is saved automatically. In case of a problem, the template is not saved, and you need to [fix it beforehand](#fixing-template-problems).
+
+   ![Template editor anatomy](img/connector-templates/create-connector-template-2.png)
+
+4. Once you've written your template, you can **publish** it by clicking on the corresponding action. You will be prompted to optionally enter a version name. This name appears in your milestone list.
+
+   ![Publishing a template](img/connector-templates/create-connector-template-3.png)
+
+5. After publishing, the template can be applied to elements of all models in the same project.
+
+   :::info Project scope
+   A connector template is always associated with a project, hence only available in the project where you added it.
+   :::
+
+### JSON editor features
+
+The JSON editor is based on the [Monaco Editor](https://microsoft.github.io/monaco-editor/). The Monaco Editor is the editor that powers VS Code. As a result, the template editor supports many familiar features such as auto-formatting, indentation support, code completion, and error highlighting. Read the [VS Code editor docs](https://code.visualstudio.com/docs/editor/editingevolved) for a full overview of features.
+
+### Versioning connector templates
+
+You can version your connector templates using [milestones](/components/modeler/web-modeler/milestones.md), similar to diagrams.
```

Comment:
I would add something like this, because this is I think a common real-life question:

`If you publish a new version of a Connector template, and the old version was already used in diagrams, the user can decide to update to the most recent version, or to stay with the old version when editing the respective diagram element (see updating [element templates](https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/using-templates/#updating-templates)).`

Reply:
Thanks, this totally makes sense!